docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  ms: &ms
+    build:
+      context: .
```

Comment:
Oh nice @FireFart, I didn't see your comment before I posted mine. I guess we were thinking the same thing.